src/screens/NoticeboardFormScreen.tsx

```diff
@@ -0,0 +1,498 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import * as Moment from 'moment';
```

Comment:
normally we import per `import moment from 'moment';`. can you do it here as well?
and change `const moment = extendMoment(Moment);` to something like `const momentRange = extendMoment(moment);` or `const extendedMoment = extendMoment(moment);` so that we know this is not the normal `moment` anymore?!

Reply:
Yes, of course I can change it.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/700/commits/6d57f01fbd1f7e857e1c42b5adad0cecca05471b